### DISCUSSION TITLE
Check for module installation

### DISCUSSION THREAD

**Discussion by angelhdzmultimedia** [nuxt/cli#224] [Ideas]
Hi! Great job with nuxi! Big fan! 🔥🤯💚

It would be nice if `nuxi module add` checked if the package is already installed and avoid the installation step, and just continue with the pushing into the nuxt.config.ts `modules: []` array.

Cheers

**Comment by pi0** [comment]
Thanks for idea! Moved to https://github.com/nuxt/cli/issues/229
